build.py

```diff
@@ -122,6 +130,10 @@ def main():
     )
     data = CHECKSUM_RE.sub('__checksum__ = "%s"' % content_checksum, data)
     data = JUMPTABLE_RE.sub("_JUMPTABLE = %s  # noqa: E501" % str(jump_table), data)
+    data = GTLD_INCLUDE_SUBDOMAINS_RE.sub(
```

Comment:
Last nit, could this be updated to render the labels in sorted order (even if they are in a set)?
That way the order in the source file doesn't depend on the order within the list.

Reply:
Thank you for the Blacken commit, I've checked ````nox -s test``` but I forgot to call ```nox -s check``` before.